src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -266,7 +268,7 @@ private void VerifyResultAndOutput((int ExitCode, string Output) result, TempDir
 
                 var result = RunCommandLineCompiler(CSharpCompilerClientExecutable, $"/shared:{serverData.PipeName} /nologo hello.cs", _tempDirectory, files, redirectEncoding: Encoding.ASCII, shouldRunOnServer: false);
                 Assert.Equal(result.ExitCode, 1);
-                Assert.Equal("hello.cs(1,1): error CS1056: Unexpected character '?'", result.Output.Trim());
+                Assert.Equal($"hello.cs(1,1): error CS1056: {string.Format(CSharpResources.ERR_UnexpectedCharacter, "?")}", result.Output.Trim());
```

Comment:
>Assert.Equal($"hello.cs(1,1): error CS1056: {string.Format(CSharpResources.ERR_UnexpectedCharacter, "?")}", result.Output.Trim()); [](start = 16, length = 130)

Please add /preferreduilang:en switch instead.

Reply:
It looks like this is applicable to all other issues in this file

---
In reply to: [168915207](https://github.com/dotnet/roslyn/pull/24460#discussion_r168915207) [](ancestors = 168915207)